libsel4vmmplatsupport/src/arch/x86/guest_boot_init.c

```diff
@@ -92,18 +93,19 @@ static void make_guest_screen_info(vm_t *vm, struct screen_info *info)
         }
         if (error) {
             ZF_LOGE("Failed to map vbe protected mode interface for VESA frame buffer. Disabling");
```

Comment:
The error message need to be coherent with the behavior

Reply:
Switched to "Not Disabling".